### ISSUE TITLE
Image alternative text missing information (& typo)

### ISSUE THREAD

**Issue by shawna-slh** [w3c/w3c-website#299]
**Describe the issue**
Image alternative text missing information.

**URL**
page: https://beta.w3.org/about/
image: https://cdn.w3.org/cms-uploads/_920xAUTO_crop_center-center_none/w3c-composition.png

**Recommended solution**
change:
> alt="concentric circles: 1 director, 50 team members, 1.5K in work group, 12K in community groups, 1.9B internet users"

to:

> alt="concentric circles: 1 director Tim Berners-Lee, 50 team members, 1.5K W3C work group members, 12K W3C community group members, 4.9B internet users"

**Comment by koalie** [comment]
Thanks, Shawn.
I filled almost all of that but not all because the alt text field in that particular module is limited to 125 characters.

**Comment by koalie** [comment]
I have created related issue #302
